lib/ruby_lsp/ruby_lsp_rails/document_symbol.rb

```diff
@@ -161,8 +161,12 @@ def handle_all_arg_types(node, message)
             name = arg_receiver.full_name if arg_receiver.is_a?(Prism::ConstantPathNode)
             next unless name
 
+            unless arg_receiver.is_a?(Prism::ConstantReadNode) || arg_receiver.is_a?(Prism::ConstantPathNode)
+              name = ":#{name}"
+            end
```

Comment:
Is it possible for it to be a `CallNode` but for the receiver to not be one of these two? That seems to be what's being guarded against

Reply:
But don't we already guard this in the conditional above? If the receiver is not a `ConstantReadNode` or `ConstantPathNode`, then `name` will be `nil` and we'll skip, no?

Reply:
Yeah, that's correct. It would be better to change the line where we're skipping, but if we do proceed with that, my question then stands about whether there's other cases worth factoring in here.